portfolio/src/main/webapp/index.html

```diff
@@ -15,38 +15,43 @@ <h1>Leah Attai</h1><h3 >Portfolio</h3>
-        	<div class = "subPar" onclick="expand('projects')"><h2>Past Projects</h2></div>
-            <div class = "subPar" onclick="expand('facts')"><h2>Fun Facts</h2></div>
-        	<div class = "subPar" onclick="expand('links')"><h2>Links</h2></div>
+        	<div class = "subPar" onclick="expand('about')"><h2 class = "link">About me</h2></div>
+        	<div class = "subPar" onclick="expand('projects')"><h2 class = "link">Past Projects</h2></div>
+            <div class = "subPar" onclick="expand('facts')"><h2 class = "link">Fun Facts</h2></div>
          </div>
         <div id= "about-container"></div>
         </br>
-        <a href= "https://www.linkedin.com/in/leah-attai-203">LinkedIn </a> </br>
-        <a href= "https://github.com/lattai">GitHub </a> </br>
+         </br>
         
-        <div id = "galleryBox">
-        <h2>Gallery</h2>
-        <div id = "gallery">
-            <div id = "gwc" class = "image"> 
-                <a target ="_blank" alt = "Girls Who Code" href = "https://www.google.com/url?sa=i&url=https%3A%2F%2Fmedium.com%2F%40GirlsWhoCode%2Fstay-curious-patience-in-coding-leah-attai-fb20cb3a6219&psig=AOvVaw2Cyv0bVxYUoMYil6qA-mgZ&ust=1590176828049000&source=images&cd=vfe&ved=0CAMQjB1qFwoTCODsjrrcxekCFQAAAAAdAAAAABAR">
-                <img src = "/images/gwc.jpeg" alt = "Girls Who Code"></a>
-            </div>
-            <div id = "csc" class = "image"> 
-               <a target ="_blank" alt = "Coding @ Cardinal Shehan" href = "https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.ctpost.com%2Flocal%2Farticle%2FBringing-girls-up-to-Scratch-13092641.php&psig=AOvVaw2Cyv0bVxYUoMYil6qA-mgZ&ust=1590176828049000&source=images&cd=vfe&ved=0CAMQjB1qFwoTCODsjrrcxekCFQAAAAAdAAAAABAV">
-                <img src = "/images/csh.jpg" alt = "Coding @ Cardinal Shehan"></a>
+        <center> <div id = "galleryBox">
+            <h2>Gallery</h2>
+            <div id = "gallery">
+                <div id = "gwc" class = "image"> 
+                    <a target ="_blank" alt = "Girls Who Code" href = "https://medium.com/@GirlsWhoCode/stay-curious-patience-in-coding-leah-attai-fb20cb3a6219">
+                    <img src = "/images/gwc.jpeg" alt = "Girls Who Code"></a>
+                </div>
+                <div id = "csc" class = "image"> 
+                    <a target ="_blank" alt = "Coding @ Cardinal Shehan" href = "https://www.ctpost.com/local/article/Bringing-girls-up-to-Scratch-13092641.php">
+                    <img src = "/images/csh.jpg" alt = "Coding @ Cardinal Shehan"></a>
+                </div>
+                <div id = "technica" class = "image">
+                    <a target ="_blank" alt = "Technica 2019" href = "https://www.gettysburg.edu/news/stories?id=854d16d8-f587-48a0-b78b-97d718e45a5c&pageTitle=Computer+Science+students+at+Gettysburg+got+it+done%21">
+                    <img src = "/images/technica.jpg" alt = "Technica 2019"></a>
+                </div>
```

Comment:
You can also do a gallery with the <table></table> tag it might be easier but this div makes for a better responsive site really great work!

Reply:
The table tag